src/reactor.h

```diff
@@ -346,6 +349,28 @@ class Reactor : public cyclus::Facility,
   }
   std::string power_name;
 
+  /////////// hybrid params ///////////
+
+  #pragma cyclus var { \
+    "uilabel": "Side Product from Reactor Plant", \
+    "default": [], \
+    "doc": "Ordered list of side product the reactor produces with power", \
+  }
+  std::vector<std::string> side_products;
+
+  #pragma cyclus var { \
+    "uilabel": "Quantity of Side Product from Reactor Plant", \
+    "default": [], \
+    "doc": "Ordered list of the quantity of side product the reactor produces with power", \
+  }
+  std::vector<double> side_product_quantity;
+
+  #pragma cyclus var {"default": 1, "doc": "This should NEVER be set manually",\
+                      "internal": True \
+  }
+  bool hybrid;
```

Comment:
if it is an internal variable, it should probably end in an underscore (ie `hybrid_`).  Also, this docstring does not explain the parameter